### PR TITLE
Added CPrint utility method to make printing to console easier. Added…

### PR DESCRIPTION
**Pull request by mjdiloreto**
Going to suggest we merge this stuff, just because I feel like you will need the import utility for whatever you are doing next, and the CPrint is pretty useful too ;)

### CODE REVIEW

File: levelSearch.lua [hunk 1]
```diff
@@ -0,0 +1,44 @@
+require("stack")
+
+function levelSucc(room)
```
Comment (2-X) [inline levelSearch.lua:3]: aptly named 😏

File: main.lua [hunk 2]
```diff
@@ -2,22 +2,48 @@ require('mobdebug').start();             -- enable debugging checkpoints
...
+
+--[[ Write str to the Isaac Console --]]
+function CPrint(str) 
+  Isaac.ConsoleOutput(str.."\n")
+end
+
+--[[ Write str to file specified by config (or hardcoded) --]]
+function log(str)
+  -- TODO change this to write to a log file.
+  CPrint(str)
```
Comment (2-X) [inline main.lua:28]: Isaac.DebugLog or something

File: main.lua [hunk 3]
```diff
@@ -2,22 +2,48 @@ require('mobdebug').start();             -- enable debugging checkpoints
...
+end
+
+--[[ Utility method for testing --]]
+function equal(expected, result)
+  -- TODO make this smart?
```
Comment (2-X) [inline main.lua:33]: you mean deeply recursive?

File: main.lua [hunk 4]
```diff
@@ -2,22 +2,48 @@ require('mobdebug').start();             -- enable debugging checkpoints
...
+  return expected == result
+end
+
+--[[ Check membership of val in list --]]
```
Comment (2-X) [inline main.lua:37]: not a fan of the block comment look tbh, super clunky

File: stack.lua [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+  CPrint(equal(nil, stack:pop()))
+  
+  stack:push(2)
+  stack:push(3)
+  CPrint(equal(3, stack:pop()))
+  CPrint(equal(2, stack:pop()))
+  CPrint(equal(nil, stack:pop()))
+end
+
+--stackTest()
```
Comment (2-X) [inline stack.lua:58]: no newline at the end of file! :'(

**Review by 2-X — APPROVED** [review]
LGTM

File: main.lua (hunk 4 above)
Comment (mjdiloreto) [reply main.lua:37]: Just want to keep it consistent if there are multiline comments required for a function

File: main.lua (hunk 3 above)
Comment (mjdiloreto) [reply main.lua:33]: I just don't know if every type supports == . I don't care about recursive, but I just don't want it to blow up if I call equal({}, 0) for example

File: main.lua (hunk 4 above)
Comment (2-X) [reply main.lua:37]: If there was any way to do multiline comments I would prefer:

--[[
does this thing
--]]

but I still think that single line comments look better

**Pull request merged**